packages/website/ts/components/top_bar/provider_display.tsx

```diff
@@ -40,21 +52,23 @@ export class ProviderDisplay extends React.Component<ProviderDisplayProps, Provi
             : 'Connect a wallet';
         const providerTitle =
             this.props.providerType === ProviderType.Injected ? injectedProviderName : 'Ledger Nano S';
+        const isProviderMetamask = providerTitle === 'Metamask';
```

Comment:
`'Metamask'` should we a constant referenced in both places where it's assigned and here.